service/interfaces.go

```diff
@@ -4,11 +4,15 @@ import (
 	"context"
 	"net/http"
 
+	"github.com/ONSdigital/dp-api-clients-go/cantabular"
+	"github.com/ONSdigital/dp-api-clients-go/dataset"
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
 )
 
 //go:generate moq -out mock/server.go -pkg mock . HTTPServer
 //go:generate moq -out mock/healthCheck.go -pkg mock . HealthChecker
+//go:generate moq -out mock/cantabular-client.go -pkg mock . CantabularClient
```

Comment:
Hmm commented out lines ?

Reply:
Yes, this is to let `go generate` know that it has to create a mock for the CantabularClient interface